index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <p><h1><strong>Ryan Gosling autobiography</strong></h1></p>
+    <p>
+        <h2><strong>Name:</strong> <em>Ryan Gosling</em></h2></p>
+        <p><strong>Age:</strong> </body> 42 years old</p>
+        <p><strong>Date of birth: <mark> November 12, 1980 </mark></strong>  </p>
+        <p><strong>Nationality:</strong> Canadian</p>
+        <p><strong>Description:</strong> Ryan Thomas Gosling (born November 12, <ins>1980</ins>) is a Canadian actor. Prominent in both independent film and major studio features of varying genres, his films have accrued a worldwide box office gross of over 1.9 billion USD. He has received various accolades, including a Golden Globe Award, and nominations for two Academy Awards and a BAFTA Award.</p>
+        <p>Born and raised in Canada, he rose to prominence at age 13 as a child actor on Disney Channel's The Mickey Mouse Club <ins>(1993–1995),</ins> and went on to appear in other family entertainment programs, including Are You Afraid of the Dark? <ins>(1995)</ins> and Goosebumps <ins>(1996)</ins>. His breakthrough film role was as a Jewish neo-Nazi in The Believer <ins>(2001)</ins>, and he gained stardom for starring in the <ins>2004</ins> romantic drama The Notebook.</p>
```

Comment:
This code doesn't look correct, heading inside paragraphs? strong inside headings?